src/store/actions.js

```diff
@@ -0,0 +1,5 @@
+import * as types from './mutation-types';
```

Comment:
`setFoo` - this file looks like it's not used?

Reply:
That's just Vuex scaffolding I didn't use and didn't have time to cleanup :)

Reply:
It's been cleaned up!